Author: Michael R. Crusoe <crusoe@debian.org>
Description: The libdeflater crate is not yet package for Debian
Forwarded: not-needed
--- bgzip.orig/Cargo.toml
+++ bgzip/Cargo.toml
@@ -36,10 +36,6 @@
 optional = true
 default-features = false
 
-[dependencies.libdeflater]
-version = "0.12.0"
-optional = true
-
 [dependencies.log]
 version = "0.4"
 optional = true
@@ -83,7 +79,6 @@
     "rayon",
 ]
 flate2 = ["dep:flate2"]
-libdeflater = ["dep:libdeflater"]
 log = ["dep:log"]
 rayon = ["dep:rayon"]
 rust_backend = [
--- bgzip.orig/Cargo.toml.orig
+++ bgzip/Cargo.toml.orig
@@ -17,7 +17,6 @@
 flate2 = ["dep:flate2"]
 rust_backend = ["flate2/rust_backend", "flate2"]
 zlib = ["flate2/zlib", "flate2"]
-libdeflater = ["dep:libdeflater"]
 rayon = ["dep:rayon"]
 log  = ["dep:log"]
 
@@ -25,7 +24,6 @@
 flate2 = { version = "1", default-features = false, optional = true }
 rayon = { version = "1.6.1", optional = true }
 log = { version = "0.4", optional = true }
-libdeflater = { version = "0.12.0", optional = true }
 thiserror = "1.0"
 
 [dev-dependencies]
